pyradiosky/skymodel.py

```diff
@@ -496,6 +496,88 @@ def read_healpix_hdf5(hdf5_filename):
     return hpmap, indices, freqs
 
 
+def write_healpix_hdf5(filename, hpmap, indices, freqs, nside=None, history=None):
+    """
+    Write a set of HEALPix maps to an HDF5 file.
+
+    Parameters
+    ----------
+    filename: str
+        Name of file to write to.
+    hpmap: array_like of float
+        Pixel values in Kelvin. Shape (Nfreqs, Npix)
+    indices: array_like of int
+        HEALPix pixel indices corresponding with axis 1 of hpmap.
+    freqs: array_like of floats
+        Frequencies in Hz corresponding with axis 0 of hpmap.
+    nside: int
+        nside parameter of the map. Optional if the hpmap covers
+        the full sphere (i.e., has no missing pixels), since the nside
+        can be inferred from the map size.
+    history: str
+        Optional history string to include in the file.
+
+    """
+    try:
+        import astropy_healpix
+    except ImportError as e:
+        raise ImportError(
+            "The astropy-healpix module must be installed to use HEALPix methods"
+        ) from e
```

Comment:
go ahead and stick a `# pragma: no cover` on this block, even when we get the minimum dependency builds working, we won't cover this because we'll skip the test entirely if `astropy-healpix` isn't installed.

Reply:
This is covered by the new test.